src/Exercism.Representers.CSharp/Options.cs

```diff
@@ -13,7 +13,12 @@ internal class Options
         [Value(2, Required = true, HelpText = "The directory to which the results will be written")]
         public string OutputDirectory { get; }
 
-        public Options(string slug, string inputDirectory, string outputDirectory) =>
-            (Slug, InputDirectory, OutputDirectory) = (slug, inputDirectory, outputDirectory);
+        [Option('s', "SuppressLogConfiguration", Required = false, HelpText = "A flag (used for unit testing) to prevent configuration of the logger")]
+        public bool SuppressLogConfiguration { get; }
```

Comment:
Naming wise, I think I would prefer `SkipLogConfiguration`, but that is just a personal preference :)

Reply:
Changed